test/conformance/testdata/conformance.yaml

```diff
@@ -186,7 +186,7 @@
 - testname: Custom Resource Definition, create
   codename: '[sig-api-machinery] CustomResourceDefinition resources [Privileged:ClusterAdmin]
     Simple CustomResourceDefinition creating/deleting custom resource definition objects
-    works  [Conformance]'
+    works [Conformance]'
```

Comment:
this requires special approvers, you better split it in a new PR for faster approval?

Reply:
Will do.

Reply:
https://github.com/kubernetes/kubernetes/pull/120959